fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/logging/LogMessageKeys.java

```diff
@@ -306,6 +306,10 @@ public enum LogMessageKeys {
     HASH_FUNCTION,
     HASH,
 
+    // Lucene
```

Comment:
I missed it, but you should update the release notes to indicate that the associated issue is being resolved.
You just replace one of the commented out lines, like:
```
* **Feature** Feature 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
```

becomes
```
* **Feature** Add lucene partitioning metadata [(Issue #2390)](https://github.com/FoundationDB/fdb-record-layer/issues/2390)
```

And then the build will put it in the actual release notes when it creates a release.